serverless.yaml

```diff
@@ -179,6 +182,10 @@ resources:
         Type: String
         Default: ${self:custom.stage}
         Description: 'The deployment stage (e.g. dev, qa, prod). Default: dev'
+      StageType:
+        Type: String
+        Default: ${self:custom.stageType}
+        Description: 'Specifies type of the deployment stage The following values are allowed( dev, prod). Default: dev'

```

Comment:
Listing allowed values is supported in CloudFormation.

https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/parameters-section-structure.html

`AllowedValues`